src/stores/api/formatters/eth1.go

```diff
@@ -88,18 +88,15 @@ func FormatEEATransaction(tx *types.SignEEATransactionRequest) (*ethtypes.Transa
 	return ethtypes.NewTransaction(uint64(tx.Nonce), *tx.To, big.NewInt(0), uint64(0), big.NewInt(0), tx.Data), privateArgs
 }
 
-func FormatEth1AccResponse(key *entities.ETH1Account) *types.Eth1AccountResponse {
+func FormatEth1AccResponse(eth1Acc *entities.ETH1Account) *types.Eth1AccountResponse {
 	return &types.Eth1AccountResponse{
-		KeyID:               key.ID,
-		Address:             key.Address,
-		PublicKey:           key.PublicKey,
-		CompressedPublicKey: key.CompressedPublicKey,
-		Tags:                key.Tags,
-		Disabled:            key.Metadata.Disabled,
-		CreatedAt:           key.Metadata.CreatedAt,
-		UpdatedAt:           key.Metadata.UpdatedAt,
-		ExpireAt:            key.Metadata.ExpireAt,
```

Comment:
as far as I understand it if a key has an expiration time the account linked to it should have the same value and therefore users should be informed about it

Reply:
`expireAt` is not implemented at the moment, we only have the concept of deleting and destroying. Deleting is equivalent to soft-delete and destroying completely destroys the keys. We do not correlate the behavior of the underlying vaults (like AWS that deletes after 30 days) in the QKM, thefore expireAt is not needed.